clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
At the time the `origCmdLine = cmdLine` alias is done, we don't even know yet whether we will need to create a new command line, i.e. whether we need a deep copy at all. And indeed, if we had used `insert` then that would modify both lists. You could also have done

```python
cmdLine = list(origCmdLine)
cmdLine.insert(0, '/showIncludes')
```

and I hope QC would accept that just as well.